adal/src/main/java/com/microsoft/aad/adal/PackageHelper.java

```diff
@@ -45,7 +45,7 @@
 /**
  * Gets information about calling activity.
  */
-class PackageHelper {
+public class PackageHelper {
```

Comment:
this should not be public

Reply:
why ?

Reply:
We do not want to support this class as a public interface. Lets talk about this in person